components/map/mapbox-map.tsx

```
@@ -519,18 +454,62 @@ export const Mapbox: React.FC<{ position?: { latitude: number; longitude: number
+          // Save current drawings before removing control
+          drawingFeatures.current = drawRef.current.getAll()
+
+          try {
+            map.current.off('draw.create', updateMeasurementLabels)
+            map.current.off('draw.delete', updateMeasurementLabels)
+            map.current.off('draw.update', updateMeasurementLabels)
+            map.current.removeControl(drawRef.current)
+            drawRef.current = null
+
+            // Clean up any existing labels
+            Object.values(polygonLabelsRef.current).forEach(marker => marker.remove())
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)